float32 val
float32 val2
---
float32 ans
float32 ans2
float32 ans3
float32 ans4

